cloud/cloud_scheduler_test.cc

```diff
@@ -174,6 +173,42 @@ TEST_F(CloudSchedulerTest, TestLongRunningJobCancel) {
   ASSERT_EQ(status.load(), JobStatus::FINISHED);
 }
 
+class CloudSchedulerRaceTest: public testing::Test {
+};
+
+// Once cloud scheduler is destructed, jobs shouldn't be erased after it's scheduled
+TEST_F(CloudSchedulerRaceTest, SkipJobEraseOnceDestructedTest) {
```

Comment:
I think you want just TEST, not TEST_F - then you don't need to declare the empty class